src/DynamicData/Cache/IKey.cs

```diff
@@ -2,6 +2,7 @@
 // Roland Pheasant licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for full license information.
 
+// ReSharper disable once CheckNamespace
```

Comment:
Would definitely want to see these annotations dropped. Dynamic data uses StyleCop and Roslynator for styling analysis, not ReSharper.

On a separate note, I'd prooobably be open to adding support for ReSharper workflows, if there's some way to do it without our team having to have ReSharper. Especially if ReSharper supports a rules config file like `.editorconfig`, so we don't have to annotate code everywhere there's rules that we regularly break.